styles/style.css

```diff
@@ -426,10 +569,20 @@ a img {
     font-size: 15px;
     line-height: 1.13em;
   }
+  /*media homes*/
+  .homes {
+    padding: 0 0 0 17px;
```

Comment:
use values that multiple of 8. Also padding: 0 0 0 17px  change to padding-left:17px. It is more suitable